app.py

```diff
@@ -17,6 +17,8 @@
 from typing import Dict, List, Optional
 from types import FrameType
 
+VERSION = "0.0.1"
```

Comment:
I've already tagged it as 0.9.0 - we're putting this into prod, so its too late to call it 0.0.1!

Reply:
Fixed version to 0.9.0.